util/wait/wait.go

```diff
@@ -0,0 +1,52 @@
+package wait
```

Comment:
What do you think about calling this package `retry`?

Reply:
Good idea!